Ginger/GingerCoreNET/DataSource/ActDSTableElement.cs

```diff
@@ -143,6 +141,28 @@ public override void Execute()
                             this.Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
                             Error = outVal;
                         }
+                        break;
+                    case eControlAction.AddRow:
+                        DataSourceTable DSTable = null;
+                        ObservableList<DataSourceTable> dstTables = DataSource.GetTablesList();
+                        foreach (DataSourceTable dst in dstTables)
+                        {
+                            if (dst.Name == DSTableName)
+                            {
+                                DSTable = dst;
+                                DSTable.DataTable = dst.DSC.GetTable(DSTableName);
+                                break;
+                            }
+                        }
+                        List<string> mColumnNames = DataSource.GetColumnList(DSTableName);
+                        DataSource.AddRow(mColumnNames, DSTable);
+                        DataSource.SaveTable(DSTable.DataTable);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['DSTable' is null on at least one execution path.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5038083)